build.gradle

```diff
@@ -27,7 +27,7 @@ plugins.withType(JavaPlugin) {
 
 ext.versions = [
         assertj      : '2.0.0',
-        guava        : '18.0',
+        guava        : '21.0',
```

Comment:
no changes? hm...

Reply:
Apparently. I updated as I wanted to have `toImmutableSet` method :)